packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -48,6 +48,7 @@ export default function ActivityItem(props) {
 ActivityItem.propTypes = {
   activity: PropTypes.object,
   avatarUrl: PropTypes.string,
+  callData: PropTypes.object,
```

Comment:
Feels kinda weird that we're passing call specific stuff into this component. Would it make more sense to construct a single `activity` object at the selector level?